def main():
    raise Exception() # throw instance

print "Start"
main()
print "End"
